### ISSUE TITLE
MA0132 should not trigger when DateTime.UtcNow is involved

### ISSUE THREAD

**Issue by NinjaCross** [meziantou/Meziantou.Analyzer#777]
### Version of the Meziantou.Analyzer NuGet package

2.0.0

### Rule Identifier

MA0132 

### Target Framework

net 6.0

### C# Language version

C# 11

### Description

This expression should not trigger the analyzer, because the `DateTime.Kind` is known to be UTC and constant.
```cs
var delta = TimeSpan.FromSeconds(2);
var next = DateTime.UtcNow +delta;
```

### Reproduction Steps

Minimal code:

```c#
var delta = TimeSpan.FromSeconds(2);
var next = DateTime.UtcNow +delta;
```

### Other information

Another example

![image](https://github.com/user-attachments/assets/92593111-6f3e-4d14-8166-8c7c68d24191)

**Comment by NinjaCross** [comment]
You are right @meziantou.
May I propose then to improve the message by adding to it this suggestion ?
